.github/workflows/ci.yml

```diff
@@ -175,3 +182,38 @@ jobs:
           version: "v${{ env.VERSION }}"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  label:
+    name: Attach automerge label
```

Comment:
can we attach the `dependency` label? it's later used when creating the release notes

Reply:
Added, some DRY was needed though.

Reply:
though now the comment needs updating, as we attach two labels :)

Reply:
or maybe we can just say `this step attaches the labels specified in labelers.yml, but only if the PR changes one file`